src/main/java/cn/nukkit/command/defaults/EnchantCommand.java

```diff
@@ -0,0 +1,53 @@
+package cn.nukkit.command.defaults;
+
+import cn.nukkit.Player;
+import cn.nukkit.command.Command;
+import cn.nukkit.command.CommandSender;
+import cn.nukkit.event.TranslationContainer;
+import cn.nukkit.item.Item;
+import cn.nukkit.item.enchantment.Enchantment;
+import cn.nukkit.utils.TextFormat;
+
+/**
+ * Created by Pub4Game on 23.01.2016.
+ */
+public class EnchantCommand extends VanillaCommand {
+
+    public EnchantCommand(String name) {
+        super(name, "%nukkit.command.enchant.description", "%commands.enchant.usage");
+        this.setPermission("nukkit.command.enchant");
+    }
+
+    @Override
+    public boolean execute(CommandSender sender, String commandLabel, String[] args) {
+        if (!this.testPermission(sender)) {
+            return true;
+        }
+        if (args.length < 2) {
+            sender.sendMessage(new TranslationContainer("commands.generic.usage", this.usageMessage));
+            return true;
+        }
+        Player player = sender.getServer().getPlayer(args[0]);
+        if (player == null) {
+            sender.sendMessage(new TranslationContainer(TextFormat.RED + "%commands.generic.player.notFound"));
+            return true;
+        }
+        int enchantId = Integer.parseInt(args[1]);
```

Comment:
Add a try... catch..., or will get an exception when args are not in number format. Just leave a note for myself.